### PR TITLE
[fix] Pass arguments to jenkins Dockerfile 

### PR DESCRIPTION
**Pull request by jgsogo**
closes https://github.com/conan-io/conan-docker-tools/issues/314

... we were not passing all the required arguments to this docker image 🤦

### CODE REVIEW

File: modern/jenkins/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,8 @@ RUN /tmp/install-openjdk-ppa.sh \
 RUN pip3 install --no-cache virtualenv
     
 RUN mkdir -p /usr/share/jenkins \ 
-    && curl -sSLo /usr/share/jenkins/slave.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${JENKINS_AGENT_VERSION}/remoting-${JENKINS_AGENT_VERSION}.jar \
+    && wget https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${JENKINS_AGENT_VERSION}/remoting-${JENKINS_AGENT_VERSION}.jar \
+    && mv remoting-${JENKINS_AGENT_VERSION}.jar /usr/share/jenkins/slave.jar \
```
Comment (uilianries) [inline modern/jenkins/Dockerfile:23]:
```suggestion
    && wget -q -O /usr/share/jenkins/slave.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${JENKINS_AGENT_VERSION}/remoting-${JENKINS_AGENT_VERSION}.jar
```

File: .ci/xenial.jenkinsfile [hunk 2]
```diff
@@ -158,7 +158,7 @@ node('Linux') {
                     String latestMaster = "${dockerUsername}/${compiler}${versionMajor}-${distro}-jenkins:${targetBranch}-latest"
 
                     dir('modern/jenkins') {
-                        buildImage(latestMaster, latestBranch, jenkinsImage, "--build-arg SOURCE_CONANIO_IMAGE=${deployImage}")
+                        buildImage(latestMaster, latestBranch, jenkinsImage, "${compilerArgs} --build-arg SOURCE_CONANIO_IMAGE=${deployImage}")
```
Comment (jgsogo) [inline .ci/xenial.jenkinsfile:161]: We were not passing the `JENKINS_AGENT_VERSION` to the dockerfile 🤦

**Comment by jgsogo** [comment]
@davidsanfal Now we are testing that the docker images with Jenkins can run 🚀

File: .ci/xenial.jenkinsfile (hunk 2 above)
Comment (uilianries) [reply .ci/xenial.jenkinsfile:161]: We are human after all :sweat_smile:

**Pull request merged**